### PR TITLE
Portaudio: added patch for sndio support

### PR DESCRIPTION
**Pull request by archfan**
I've converted the patches from FreeBSD / OpenBSD for portaudio. This should enable SNDIO support.

See: https://svnweb.freebsd.org/ports/head/audio/portaudio/files/

### CODE REVIEW

**Comment by maxice8** [comment]
I can't test sndio.

@Duncaen

File: srcpkgs/portaudio/template [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-revision=1
+revision=2
 wrksrc=portaudio
 build_style=gnu-configure
-configure_args="--enable-cxx --with-jack"
+configure_args="--enable-cxx --with-jack --with-sndio"
 hostmakedepends="automake libtool pkg-config"
-makedepends="alsa-lib-devel jack-devel"
+makedepends="alsa-lib-devel jack-devel sndio-devel"
 homepage="http://www.portaudio.com"
```
[the remark this answers is not in the document]
Comment (archfan) [reply srcpkgs/portaudio/template]: Will do.

**Comment by maxice8** [comment]
Squash all into one commit

**Comment by archfan** [comment]
No idea how to squash. I've read a few guides here and there but I don't get the result I want.

**Comment by Johnnynator** [comment]
> No idea how to squash. I've read a few guides here and there but I don't get the result I want. I thought maintainers are able to squash commits as well.

`git rebase -i HEAD~3` And then put a `f` instead of `pick` in front of the last two commits.

**Comment by archfan** [comment]
@#$@#$! I'll open a new pull request.

**Pull request closed without merging**